active/NavigationView/TopPadding/TopPaddingMargins.md

```diff
@@ -0,0 +1,44 @@
+
+# Background
+
+Some app developers choose to extend their app content into the title bar using the `CoreApplication.GetCurrentView().TitleBar.ExtendViewIntoTitleBar = true;` API. 
+When developers do so, the shell continues to reserve a visually transparent title bar for window drag operations. 
+This has the effect of making UI elements in the top 32 pixels of the app window impossible to click with pointer input. [NavigationView](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview) draws a back button and/or its hamburger button near the top of the control, potentially underneath the transparent title bar. 
+To help app developers optimize interactivity and avoid frustrated users, starting in RS5 we added logic to automatically provide top padding and avoid hit test occlusion by the system title bar.
+
+While this feels like the correct default behavior, we do not currently provide a mechanism for app developers to opt out of this automatic padding. 
+App devs can `Window.Current.SetTitleBar` to their prefered UIElement, thus removing the hit test occlution by the transparent title bar across the entire window. 
+And although this resolves the interaction problem, NavigationView's padding remains, providing now unwanted whitespace.
+
+
+# Description
+
+App developers have greater control over [customizing](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview#navigation-view-customization) the margins and padding near the top of their instance of NavigationView. 
+
+# Navigation view customization
+
+
+## Top whitespace
+Some apps choose to [customize their window's title bar](https://docs.microsoft.com/en-us/windows/uwp/design/shell/title-bar), potentially extending their app content into the title bar area. 
+When NavigationView is the root element in apps that extend into the title bar, the control automatically adjusts the position of its interactive elements to prevent overlap with [the draggable region](https://docs.microsoft.com/en-us/windows/uwp/design/shell/title-bar#draggable-regions). 
+If your app handles the draggable region of the title bar and you would prefer to have the back and menu buttons draw closer to the top of your app window, set `IsTitleBarPaddingEnabled` to False.
+
+````Xaml
+<muxc:NavigationView x:Name="NavView" IsTitleBarPaddingEnabled="False">
+````
```

Comment:
What's the guidance/Example if IsTitleBarPaddingEnabled=False? Backbutton may be not clickable, we need to workaround the problem. I suggest we provide an example here which will make BackButton clickable even if it's on titlebar area.

Reply:
There's an [article](https://docs.microsoft.com/en-us/windows/uwp/design/shell/title-bar) which explains title bar customization in detail. I don't want to duplicate the same information in the NavigationView article.

Personally, I don't recommend extending into the title bar. That was our guidance when NavigationView's Expanded pane used acrylic, which it doesn't anymore. The title bar provides a variety of user benefits (familiarity and consistency with other apps, color hints about whether the app has focus, a place for the app title) and apps shouldn't undertake the decision to overwrite it lightly. We still don't have good design guidance of where the app title ought to appear in a NavigationView that's drawing into the title bar.

I think that documenting how to perform this customization encourages apps to extend into the title bar. And I would prefer not to encourage this.